Makefile

```diff
@@ -229,6 +229,7 @@ e2e-test/image/e2e/chaos-mesh: helm/chaos-mesh
 # $(IMAGE_$(4)_PROJECT): the project name of the image, by default it is `pingcap`
 # $(IMAGE_$(4)_REGISTRY): the registry name of the image, it will override `DOCKER_REGISTRY`
 # $(IMAGE_$(4)_TAG): the tag of the image, it will override `IMAGE_TAG`
+# $(IMAGE_$(4)_BUILD): whether the image should be pulled rather than build locally, by default it is `1`.
```

Comment:
```suggestion
# $(IMAGE_$(4)_BUILD): whether the image should be built locally rather than pulled from remote registry, by default it is `1`.
```

https://github.com/YangKeao/chaos-mesh/blob/2c5b371e5408afe9bb4f9ef01de41d632c28d00e/Makefile#L260-L263

if I understand correctly, for `IMAGE_$(4)_BUILD`
- `0` means pulling the image from a remote registry;
- `1` means building the image locally.

Reply:
Nice Catch!